src/Framework/Runner/DataTypes.cs

```diff
@@ -361,12 +391,41 @@ public class TestData : ITestData
         public virtual string ModelPath { get; set; }
 
         [XmlIgnore]
-        public bool IsNodeExpanded { get; set; }
+        public bool IsNodeExpanded
+        {
+            get
+            {
+                return _isNodeExpanded;
+            }
+            set
+            {
+                if (_isNodeExpanded != value)
+                {
+                    _isNodeExpanded = value;
+                    OnPropertyChanged();
+                }
+            }
+        }
 
         [XmlIgnore]
         public bool ModelExists
         {
-            get { return ModelPath != null && File.Exists(ModelPath); }
+            get
+            {
+                bool modelExists = false;
+
+                try
+                {
+                    modelExists = (ModelPath != null) && File.Exists(ModelPath);
+                }
+                catch
+                {
+                    // Nothing to do, just say the model isn't there (probably a wildcard in the file name
```

Comment:
should this be logging something?

Reply:
I don't think so, this is used by the UI to highlight tests that have a missing model, so there is a feedback loop there already. the reason I had to change it was because the path now can be a wildcard which `File.Exists` doesn't like